awx/ui_next/src/screens/Job/Job.jsx

```diff
@@ -99,7 +99,14 @@ class Job extends Component {
       return (
         <PageSection>
           <Card className="awx-c-card">
-            <ContentError error={contentError} />
+            <ContentError error={contentError}>
+              {contentError.response.status === 404 && (
+                <span>
+                  {i18n._(`The page you requested could not be found.`)}{' '}
+                  <Link to="/jobs">{i18n._(`View all Jobs.`)}</Link>
```

Comment:
If we're going to link, should we do it for all content errors and not just 404s?